ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -489,9 +488,13 @@ objects:
         done
 
         # wait for installer create cluster and return its status
+        # disable errexit on pipeline failures for the wait command
+        set +e
```

Comment:
Note to self: the `RC` handling which we're fixing up here was born with the template in #8271.